docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -2,14 +2,13 @@
 Hive connector with Azure Storage
 =================================
 
-The :doc:`hive` can be configured to query
-Azure Standard Blob Storage and Azure Data Lake Storage Gen2 (ABFS). Azure Blobs
-are accessed via the Windows Azure Storage Blob (WASB). This layer is built on
-top of the HDFS APIs and is what allows for the separation of storage from the
-cluster.
-
-Trino supports both ADLS Gen1 and Gen2. With ADLS Gen2 now generally available,
-we recommend using ADLS Gen2. Learn more from `the official documentation
+The :doc:`hive` can be configured to use Azure Data Lake Storage Gen2 to map big
```

Comment:
I'm not sure what this first sentence means. It sounds like marketing speak.

After digging through the Azure docs, it seems that ADLS Gen2 is built on Blob Storage, but the main ADLS front page doesn't mention that at all, so that seems like an implementation detail that we don't need to mention here.

Let's write it like this with a link:

> The :doc:`hive` can be configured to use [Azure Data Lake Storage (Gen2)](https://azure.microsoft.com/en-us/products/storage/data-lake-storage). Trino supports Azure Blob File System (ABFS) to access data in ADLS Gen2.